corehq/apps/app_manager/app_strings.py

```diff
@@ -90,6 +89,15 @@ def maybe_add_index(text):
                                 module, detail_type, column, index, property
                             ), trans(values)
 
+            # To list app strings for properties used as sorting properties only
+            if detail.sort_elements:
+                column_fields = [column.field for column in detail.get_columns()]
+                only_sort_options = [se for se in detail.sort_elements if se.field not in column_fields]
+                for sort_element in only_sort_options:
+                    if bool(sort_element.header):
+                        column = create_temp_sort_column(sort_element, len(set(detail.get_columns())) - 1)
```

Comment:
Made some changes to ensure same order but there was a challenge in extracting out a util method since there is more happening [here](https://github.com/dimagi/commcare-hq/pull/12523/files#diff-5fb1ec6df6f30bf5f8223273482f0704R369). 
I am not sure if there is a purpose of doing [this](https://github.com/dimagi/commcare-hq/pull/12523/files#diff-5fb1ec6df6f30bf5f8223273482f0704R375). Is that needed or can be removed?